spec/dummy/db/migrate/3_create_payload_hypertable.rb

```diff
@@ -2,10 +2,10 @@
 
 class CreatePayloadHypertable < ActiveRecord::Migration[Rails.version[0..2]]
   def up
-    execute 'CREATE SEQUENCE payload_id_seq'
+    enable_extension 'pgcrypto'
```

Comment:
should this go in its own migration?

Reply:
Sure, I can move this to a separate migration 💪